tests/validations/test_metrics.py

```diff
@@ -345,145 +342,6 @@ def test_derived_metric() -> None:  # noqa: D
     check_error_in_issues(error_substrings=expected_substrings, issues=build_issues)
 
 
-def test_where_filter_validations_happy(  # noqa: D
```

Comment:
Did you change these tests or just move them? With all the logic moved around in this PR, it makes it hard to tell what logic was changed vs. just moved, so I just want to make sure the old tests are still passing!

Reply:
(resolved below :-) )